File: URI-cpp/1020.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{ 
    int days,years,months,days1;

    cin >> days;

    years = days/365;
    months = (days%365)/30;
    days = (days%365)%30;

    cout << years << " ano(s)" << endl;
    cout << months << " mes(es)" << endl;
    cout << days << " dia(s)" << endl;

    return 0;
}

File: URI-c#/1020.cs
using System;

class Program
{
    static void Main()
    {
        int days = Convert.ToInt32(Console.ReadLine()), years, months;
        years = days / 365;
        months = (days % 365) / 30;
        days = (days % 365) % 30;

        Console.WriteLine("{0:D} ano(s)\n{1:D} mes(es)\n{2:D} dia(s)",years,months,days);
    }
}